katas/design-patterns/constructor-pattern/carexample.js

```diff
@@ -0,0 +1,13 @@
+function Car(model, year, miles) {
```

Comment:
please add a space before the Car to maintain the same consistency as eslint

Reply:
Daniel, eslint ask for space before the braces, not in that parentesis